<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModuleScreen</class>
 <widget class="QDialog" name="ModuleScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tab_modulos">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>990</width>
     <height>581</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>203</red>
         <green>203</green>
         <blue>203</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>203</red>
         <green>203</green>
         <blue>203</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>203</red>
         <green>203</green>
         <blue>203</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::West</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_modulos_carros">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Carros</string>
    </attribute>
    <widget class="QTabWidget" name="tab_carros">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>10</y>
       <width>950</width>
       <height>560</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_carros_registros">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Light">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>203</red>
            <green>203</green>
            <blue>203</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Light">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>203</red>
            <green>203</green>
            <blue>203</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Light">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>203</red>
            <green>203</green>
            <blue>203</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Registros</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>10</y>
         <width>251</width>
         <height>511</height>
        </rect>
       </property>
       <property name="title">
        <string>Filtro</string>
       </property>
       <widget class="QPushButton" name="btn_carros_registros_filtro_aplicar">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>110</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Aplicar</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>231</width>
          <height>24</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_carros_registros_filtro_status">
           <property name="text">
            <string>Disponibilidade:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cbx_carros_registros_filtro_status">
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Alugado</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Disponivel</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Manutenção</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>230</width>
          <height>24</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_carros_registros_filtro_modelo">
           <property name="text">
            <string>Modelo:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="txt_carros_registros_filtro_modelo"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QTableWidget" name="tbl_carros_registros">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>661</width>
         <height>511</height>
        </rect>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::NoPen</enum>
       </property>
       <property name="cornerButtonEnabled">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>93</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>40</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Status</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Modelo</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Cor</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Placa</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Chassi</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Km</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_carros_cadastro">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Light">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>203</red>
            <green>203</green>
            <blue>203</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Light">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>203</red>
            <green>203</green>
            <blue>203</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Light">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>203</red>
            <green>203</green>
            <blue>203</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Cadastro</string>
      </attribute>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>911</width>
         <height>48</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>890</width>
          <height>50</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_carros_cadastro_idatual">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>24</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>ID Atual:</string>
           </property>
           <property name="indent">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="lbl_carros_cadastro_id">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>24</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>-1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="indent">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_3">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>24</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>77</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="lbl_carros_cadastro_kmatual">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>Km:</string>
           </property>
           <property name="indent">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLCDNumber" name="lcd_carros_cadastro_kmatual">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>410</y>
         <width>610</width>
         <height>120</height>
        </rect>
       </property>
       <property name="title">
        <string>Observação</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTextEdit" name="rtx_carros_cadastro_observacao">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>310</width>
         <height>130</height>
        </rect>
       </property>
       <property name="title">
        <string>Aparência</string>
       </property>
       <widget class="QLabel" name="lbl_carros_cadastro_cor">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>25</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Cor:</string>
        </property>
       </widget>
       <widget class="Line" name="line_2">
        <property name="geometry">
         <rect>
          <x>19</x>
          <y>69</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QComboBox" name="cbx_carros_cadstro_cor">
        <property name="geometry">
         <rect>
          <x>119</x>
          <y>59</y>
          <width>180</width>
          <height>20</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Azul</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Verde</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vermelho</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cinza</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Preto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Branco</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Laranja</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rosa</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ciano</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Marinho</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vinho</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="lbl_carros_cadastro_status">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>27</y>
          <width>37</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Satus:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cbx_carros_cadstro_status">
        <property name="geometry">
         <rect>
          <x>119</x>
          <y>29</y>
          <width>180</width>
          <height>20</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Alugado</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Disponivel</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Manutenção</string>
         </property>
        </item>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>19</x>
          <y>39</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLineEdit" name="txt_carros_cadastro_modelo">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>90</y>
          <width>230</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Modelo</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_carros_cadastro_cor_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>50</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Modelo:</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>310</width>
         <height>137</height>
        </rect>
       </property>
       <property name="title">
        <string>Técnico</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_carros_cadastro_cor_3">
          <property name="text">
           <string>Placa:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="lbl_carros_cadastro_cor_4">
          <property name="text">
           <string>Chassi:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="txt_carros_cadastro_placa">
          <property name="text">
           <string>Placa</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="txt_carros_cadastro_chassi">
          <property name="text">
           <string>Chassi</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lbl_carros_cadastro_cor_6">
          <property name="text">
           <string>Km:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="lbl_carros_cadastro_cor_5">
          <property name="text">
           <string>Km Rev:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLineEdit" name="txt_carros_cadastro_kmatual">
          <property name="text">
           <string>Km</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="txt_carros_cadastro_kmrev">
          <property name="text">
           <string>Km Rev</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="btn_carros_cadastro_gravar">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>500</y>
         <width>100</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Gravar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_carros_cadastro_novo">
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>500</y>
         <width>100</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Novo</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_carros_cadastro_excluir">
       <property name="geometry">
        <rect>
         <x>830</x>
         <y>500</y>
         <width>100</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Excluir</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_modulos_clientes">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Clientes</string>
    </attribute>
    <widget class="QTabWidget" name="tab_clientes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>950</width>
       <height>560</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_clientes_registros">
      <attribute name="title">
       <string>Registros</string>
      </attribute>
      <widget class="QTableWidget" name="tbl_clientes_registros">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>661</width>
         <height>511</height>
        </rect>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::NoPen</enum>
       </property>
       <property name="cornerButtonEnabled">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>93</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>40</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Status</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Modelo</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Cor</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Placa</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Chassi</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Km</string>
        </property>
       </column>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>10</y>
         <width>241</width>
         <height>511</height>
        </rect>
       </property>
       <property name="title">
        <string>Filtros</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_clientes_cadastros">
      <attribute name="title">
       <string>Cadastros</string>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_modulos_contratos">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Contratos</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_modulos_financeiro">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Financeiro</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_cadastros_funcionarios">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Funcionarios</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
